packages/strapi-plugin-upload/middlewares/upload/index.js

```diff
@@ -23,6 +24,7 @@ module.exports = strapi => ({
       strapi.app.onerror(err);
     });
 
-    strapi.router.get('/uploads/(.*)', range, koaStatic(staticDir, { defer: true }));
+    const localServerConfig = _.get(strapi.plugins.upload.config, 'localServer', {});
```

Comment:
To get the most of _.get and to prevent failing if config.localServer was set to null. (null is a value for _.get)
const localServerConfig = _.get(strapi, 'plugins.upload.config.localServer') || {}